Vagrantfile

```diff
@@ -0,0 +1,38 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/bionic64"
+  config.vm.synced_folder ".", "/vagrant", type: "smb"
```

Comment:
Isn't this the default?

Reply:
That's what I thought but for some reason it didn't work unless I had this...